class CheckerError(Exception):
    pass


class CheckerFailedError(Exception):
    pass
